tests/test_uwsgi_plugin.py

```diff
@@ -0,0 +1,62 @@
+from testing.sandbox import install_fake_uwsgi
+install_fake_uwsgi()
```

Comment:
@asottile I'd really like to simply use uwsgi, in fact we don't even *need* the module, we mock out everything since we're just testing logic specific to the plugin functions. However we need to be able to `import uwsgi`, but that would require running the tests themselves inside a uwsgi application server since the uwsgi python plugin injects the module into the environment at run time after initializing the interpreter. I'll see about wrapping the import inside a pytest startup/teardown.